oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/forms/components/ArrayFieldItem/ArrayFieldItem.jsx

```diff
@@ -41,10 +50,11 @@ export const ArrayFieldItem = ({
           />
         ) : (
           <Button
-            style={{ marginTop: "1.75rem" }}
             aria-label={i18next.t("Remove field")}
             className="close-btn"
+            type="button"
             icon
+            id={indexPath}
```

Comment:
Element id should be unique across DOM, index number isn't

![Screenshot 2024-06-07 at 12 30 52](https://github.com/oarepo/oarepo-ui/assets/646743/c5479ac0-7a32-4ce8-be47-48d0ee40adf8)